samples/MvcApp/App_Start/Startup.Auth.cs

```diff
@@ -24,7 +27,23 @@ public void ConfigureAuth(IAppBuilder app)
             app.UseCookieAuthentication(new CookieAuthenticationOptions
             {
                 AuthenticationType = DefaultAuthenticationTypes.ApplicationCookie,
+                CookieName = ".AspNet.ApplicationCookie",
+                CookieSameSite = SameSiteMode.Lax,
+                SlidingExpiration = true,
+                ExpireTimeSpan = TimeSpan.FromMinutes(120),
                 LoginPath = new PathString("/Account/Login"),
+                TicketDataFormat = new AspNetTicketDataFormat(
+                    new DataProtectorShim(
+                        // This directory is used to share dataprotection keys between MvcApp and MvcCoreApp
+                        DataProtectionProvider.Create(new DirectoryInfo("C:\\keyDirectory"),
+                        (builder) =>
+                        {
+                            builder.SetApplicationName("CommonMvcAppName");
+                        })
+                        .CreateProtector(
+                            "Microsoft.AspNetCore.Authentication.Cookies.CookieAuthenticationMiddleware",
+                            "Identity.Application",
+                            "v2"))),
```

Comment:
I think there was a change to chunking format for large cookies that's not covered here.
https://github.com/dotnet/aspnetcore/blob/de68feabb4dc84d2d5fbcfd7b3ed0b08f53216fa/src/Shared/ChunkingCookieManager/ChunkingCookieManager.cs#L42
https://github.com/aspnet/AspNetKatana/blob/e003703a8bbf003f56ad160a8d034d9560639bc8/src/Microsoft.Owin/Infrastructure/ChunkingCookieManager.cs#L46
https://github.com/dotnet/aspnetcore/blob/release/2.1/src/Security/Interop/src/ChunkingCookieManager.cs

Reply:
One option would be to service Cookies.Interop with the recommended improvements. It's a stand-alone 2.1 package.

Reply:
I think we should just go with adding whatever API we need to the adapters in the short term, can always move it somewhere later if there's demand

Reply:
If anything I think we should have adapters take over anything that used to be in cookie interop, since that was only built for migration scenarios anyways

Reply:
@mjrousos where do you think an appropriate place for the UseSharedCookie helper to live in adapters, the MVC app isn't currently using any adapter code right now right?

Reply:
So extending IAppBuilder would require pulling in Owin for adapters I think to add the extension to IAppBuilder, is that something we are okay doing?

Reply:
Okay I see what I was missing, VS wasn't showing me any of the .Framework 472 files this makes more sense to me now

Reply:
That's a good point about extending IAppBuilder requiring Owin. If we can come up with a nice surface area that extends `ISystemWebBuilder` that may be the better way to go. My only concern is that it may be difficult to come up with an API that has the user configure cookie sharing on the ISystemWebBuilder while configuring cookies on IAppBuilder.

An alternative would be to have a separate project (Microsoft.AspNetCore.SystemWebAdapaters.Owin or something like that) where we add IAppBuilder extensions for things like this. Then users who care about that scenario can reference that package (and they will already have OWIN dependencies) and others can ignore it.